python-diamondpurvis/hello-diamond.py

```diff
@@ -0,0 +1,19 @@
+#Hard Coded
```

Comment:
Change filename by hello-DiamondPurvis.py and move it to the python directory (instead of python-diamondpurvis)

Reply:
updated